### DISCUSSION TITLE
Is multiple repositories with more than 16 stars will be awarded a single Starstruck badge ? Not all repositories were considered.

### DISCUSSION THREAD

**Discussion by lakshmaji** [community/community#102949] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have multiple repositories (`5`) over `16 stars`. But in the profile section only one `Starstruck` badge. 
Is it anticipated behaviour ? 

**Bug**: This is a bug where the `Starstruck` needs to display a `x4` label next to badge along with links to repositories which has over 16 stars.

Note: The `Pull Shark` badge is displaying the correct label `x3` with links to pull requests.

**Comment by MrUprizing** [comment]
Edit profile and add Starstruck

**Reply by lakshmaji** [reply]
Thanks for the response! However, GitHub Achievements are awarded or added to profile based on the number of stars received in a repository. Where I have repositories with stars but the badges are not added.
